### ISSUE TITLE
Error when parsing `export hello from './hello'`

### ISSUE THREAD

**Issue by leeching** [eslint/eslint#4914]
```js
// hello.js
export const hello = () => console.log('hello');

// index.js
export hello from './hello';
```
When eslinting, error occurs.
> error  Parsing error: Unexpected identifier

How can I config .eslintrc to escape the error? Thanks.

**Comment by eslintbot** [comment]
@leeching Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by gyandeeps** [comment]
Did you intend to write `import` and not `export`.
Like this
```js
// index.js
import hello from './hello';
```

**Comment by michaelficarra** [comment]
You have a syntax error. You probably meant

```js
export {hello} from './hello';
```

**Comment by leeching** [comment]
Sorry, I made a mistake. It should be
```js
// hello.js
export default function () { console.log('hello') };
// index.js
export hello from './hello';
```
But the error remains.

**Comment by btmills** [comment]
@leeching try @michaelficarra's suggestion. Your code right now has a syntax error.

**Comment by leeching** [comment]
@btmills I see, thanks.

**Comment by tleunen** [comment]
What about the default export? Why isn't this supported by eslint?

**Comment by btmills** [comment]
@tleunen try `export { default as hello } from './hello';`. The shorthand form isn't yet standardized (stage 2 right now I think? I'm on my phone or I'd check).

**Comment by tleunen** [comment]
Weird.. I tried it before posting, and it didn't work. I tried it again, and it works now.. Well, thanks :)
Seems to be still stage 1: https://github.com/leebyron/ecmascript-export-default-from

Anyway.. thanks @btmills!
